addons/website_slides/models/slide_channel.py

```diff
@@ -152,7 +152,7 @@ def _get_default_enroll_msg(self):
     # description
     name = fields.Char('Name', translate=True, required=True)
     active = fields.Boolean(default=True, tracking=100)
-    description = fields.Html('Description', translate=True, help="The description that is displayed on top of the course page, just below the title")
+    description = fields.Html(translate=True, help="The description that is displayed on top of the course page, just below the title")
```

Comment:
I removed it in order to create a page inside of the channel's form view that has a label `description` on the page title and not inside of it, since there is only this field that is present in the `description` page.

![Description](https://user-images.githubusercontent.com/33598078/134004640-2b4a41ad-9ffc-4fae-8823-eeb02615f42f.png)

Reply:
`nolabel="1"` did work in that case ?
It seems more like a view change, I would not remove the label of the field on the python side, but hide it on the XML side

Reply:
I tested it and it worked so I'm replacing the string of `description`.